src/tribol/common/Parameters.hpp

```diff
@@ -118,11 +120,11 @@ enum ContactMethod // all mortar methods go first
  */
 enum ContactModel
```

Comment:
Is this to define the tangential contact response?  What's the difference between `NO_CONTACT` and `FRICTIONLESS`?

Reply:
The contact model is used to specify some constitutive response across an interface. For now, this will generally be in the tangential direction. That said, if for some reason a host code wants to lose contact in a coupling scheme, this would be the place to specify no contact, or no interaction/physics. There is not a use case for this right now, and there may never be one, but for now I'll keep this. It would ease a host-code integration where they can keep the coupling scheme and all tribol calls (e.g. not guarded), but switch to no contact for whatever reason.